core/src/main/java/org/apache/iceberg/MetricsConfig.java

```diff
@@ -46,6 +47,35 @@ public static MetricsConfig getDefault() {
     return spec;
   }
 
+  public static Map<String, String> updateProperties(Map<String, String> props, List<String> deletes,
+                                                     Map<String, String> renames) {
+    if (!props.keySet().stream().anyMatch(key -> key.startsWith(TableProperties.METRICS_MODE_COLUMN_CONF_PREFIX))) {
```

Comment:
nit: shall we do a static import for `METRICS_MODE_COLUMN_CONF_PREFIX`? We have two such imports in this class already.